### ISSUE TITLE
Tailwind styles don't work out of the box in storybook.

### ISSUE THREAD

**Issue by r0zar** [pankod/superplate-core-plugins#196]
As stated, and initially opened in PR: https://github.com/pankod/superplate/issues/231

**Comment by omeraplak** [comment]
@r0zar Hi,
Would you like to open a PR that will solve this issue?

**Comment by r0zar** [comment]
Yeah I can take a stab at it. I can try and boil down the solution to the minimal change. can you drop a link for me where it makes sense to start reading the code?

**Comment by omeraplak** [comment]
cool, you can check out [development docs](https://pankod.github.io/superplate/docs/development/how-it-works)

**Comment by r0zar** [comment]
hmmm... after second though I have pass on fixing this. I'm pretty busy and it would be a bad idea to commit to it since I don't have the time to back it up. I do apologize.
